### PR TITLE
i18n - extracting new, pulling from tx

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
[the remark this answers is not in the document]
Comment (chris1984) [reply LICENSE:1]: I ran the steps in the branching doc and got that. I can remove it if you want

[the remark this answers is not in the document]
Comment (ianballou) [reply LICENSE:1]: That is really strange, I'm not sure which part would throw in a new license. But yeah let's keep the old license.

[the remark this answers is not in the document]
Comment (ianballou) [reply LICENSE:1]: What's funnier is that this is the wrong sorta license anyway! Adds to the mystery xD

**Comment by chris1984** [comment]
@ianballou removed the invalid license, so it's ready to review again

**Review by ianballou — APPROVED** [review]
Looks good :+1:

**Pull request merged**